ci/requirements-optional-conda.txt

```diff
@@ -1,4 +1,4 @@
-beautifulsoup4
+beautifulsoup4>=4.2.1
```

Comment:
I think you need to change this:

```
ci/requirements-2.7_COMPAT.pip:beautifulsoup4==4.2.0
```

Reply:
Figured as such. Without knowing the full impact, is that req file for a Travis build that tests the 4.2.0 incompatibility (ref #4259). If so do we even need this?

Reply:
you can just change it to the new minimum version 4.2.1 (we have 1 other build which is pinned as well). always like to test 1 or 2 builds for the min.